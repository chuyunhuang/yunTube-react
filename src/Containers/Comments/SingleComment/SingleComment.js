import React from 'react';
import './SingleComment.scss';
import { Button, Image } from 'semantic-ui-react';
import { Rating } from '../../../Components/Rating/Rating';

export function SingleComment(props) {
  return (
    <div className='single-comment'>
      <Image className="user-image" src='http://via.placeholder.com/48x48' circular />
      <div>
        <div className="user-name">User Name</div>
        <span>Comment texts</span>
        <div className="comment-actions">
          <Rating ikeCount={1} /><Button size='mini' compact>Reply</Button>
        </div>
      </div>
    </div>
  )
}