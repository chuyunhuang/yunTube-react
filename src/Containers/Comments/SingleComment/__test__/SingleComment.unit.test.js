import React from 'react';
import { shallow } from 'enzyme';
import { SingleComment } from '../SingleComment';

describe('SingleComment', () => {
  test('renders SingleComment', () => {
    const wrapper = shallow(<SingleComment />)
    expect(wrapper).toMatchSnapshot();
  });
});