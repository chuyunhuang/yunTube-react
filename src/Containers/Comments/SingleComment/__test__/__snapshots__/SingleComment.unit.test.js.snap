// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleComment renders SingleComment 1`] = `
<div
  className="single-comment"
>
  <Image
    as="img"
    circular={true}
    className="user-image"
    src="http://via.placeholder.com/48x48"
    ui={true}
  />
  <div>
    <div
      className="user-name"
    >
      User Name
    </div>
    <span>
      Comment texts
    </span>
    <div
      className="comment-actions"
    >
      <Rating
        ikeCount={1}
      />
      <Button
        as="button"
        compact={true}
        size="mini"
      >
        Reply
      </Button>
    </div>
  </div>
</div>
`;
